### PR TITLE
shalinee | stage | GIDK-1d6w9k | added Id on checkbox and input

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/theme/tax-control/tax-control.component.html [hunk 1]
```diff
@@ -6,11 +6,11 @@
 
         <div class="multi-select adjust" id="tax-control-multi-select">
 
-            <input #taxInputElement type="text" readonly name="sum" *ngIf="!maskInput"
+            <input #taxInputElement type="text" id="tax-contorl-input" readonly name="sum" *ngIf="!maskInput"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/tax-control/tax-control.component.html:9]: fix spelling of control everywhere

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/tax-control/tax-control.component.html [hunk 2]
```diff
@@ -6,11 +6,11 @@
 
         <div class="multi-select adjust" id="tax-control-multi-select">
 
-            <input #taxInputElement type="text" readonly name="sum" *ngIf="!maskInput"
+            <input #taxInputElement type="text" id="tax-control-input" readonly name="sum" *ngIf="!maskInput"
                 class="form-control cursor-pointer text-right taxInput" [(ngModel)]="taxTotalAmount"
                 (focus)="handleInputFocus()" style="padding-right:28px !important;" (blur)="taxInputBlur($event)" />
 
-            <input #taxInputElement type="text" readonly name="sum" *ngIf="maskInput"
+            <input #taxInputElement type="text"  id="tax-contorl-input-mask" readonly name="sum" *ngIf="maskInput"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/tax-control/tax-control.component.html:13]: fix spelling of control here as well.

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/theme/tax-control/tax-control.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/theme/tax-control/tax-control.component.html]: Done

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/tax-control/tax-control.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/theme/tax-control/tax-control.component.html]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**